### ISSUE TITLE
Icon Request: Generic DNS, Load Balancer, GSLB, F5, Citrix

### ISSUE THREAD

**Issue by HariSekhon** [mingrammer/diagrams#883]
Feature request to add the following icons (I know you can use Custom and download icons off the internet, I just think some of these are quite big ones to miss in architecture):

- generic DNS, Load Balancer and GSLB icons under `diagrams.generic`
- F5 and Citrix under `diagrams.onprem.network`

**Comment by fawaf** [comment]
Very much +1

**Comment by fawaf** [comment]
Having a generic "internet" or "cloud" would also be great

**Comment by fawaf** [comment]
Same for a generic "service"

**Comment by mortymacs** [comment]
+1

**Comment by bufo333** [comment]
I agree this is almost perfect for network diagrams except no F5, Cisco, Juniper, Infoblox, etc etc. At the very least it could use some nice brand neutral routers switches and cloud icons.
